addons/resource.language.en_gb/resources/strings.po

```diff
@@ -24192,7 +24192,7 @@ msgstr ""
 #. Help text of setting "Allow use DXVA Video Super Resolution" with label #13417
 #: system/settings/settings.xml
 msgctxt "#39194"
-msgid "Enables advanced DXVA upscaler using NVIDIA \"RTX Video Super Resolution\" or \"Intel Video Super Resolution\".[CR]Used when the video source is 1920x1080 or less and the source resolution is lower than the display resolution.[CR]Only available on specific hardware: NVIDIA RTX 20xx, 30xx, 40xx and above, Intel Arc A770, A750."
+msgid "Enables advanced DXVA upscaler using NVIDIA \"RTX Video Super Resolution\" or \"Intel Video Super Resolution\".[CR]Used when the video source is 2560x1440 or less and the source resolution is lower than the display resolution.[CR]Only available on specific hardware: NVIDIA RTX 20xx, 30xx, 40xx and above, Intel Arc A770, A750."
```

Comment:
2560x1440 or less (NVidia) or 1920x1080 or less (Intel)

Reply:
This makes no sense, we can't be updating this string with every new Intel or NVIDIA driver release. There's also no guarantee that this is common for all NVIDIA HW at all. e.g. RTX 4070 can support HDR but RTX 2060 only SDR. This string should only be generic and indicative. I'm removing this change.

Reply:
Users should check the Intel / Nvidia support website to see if their specific graphics card model and x.xx.xx video driver version support Video Super Resolution with x resolution and HDR or 10-bit SDR, etc.

Reply:
Yes making the message more generic is good, don't remove the change altogether. 
"VSR is an evolving technology, consult the manufacturers support for up to date information on the supported graphics adapters, resolutions and formats." or something like it.